src/Features/Core/Portable/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.ProjectState.cs

```diff
@@ -223,6 +224,8 @@ public bool IsEmpty(DocumentId documentId)
                 }
 
                 await SerializeAsync(serializer, project, result.ProjectId, _owner.NonLocalStateName, result.Others).ConfigureAwait(false);
+
+                AnalyzerABTestLogger.LogProjectDiagnostics(project, result);
```

Comment:
as I wrote in the description, if we remove error count for document, we get nothing for people who has FSA off which is default for C#.